# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1234426006 18000
#      Thu Feb 12 03:06:46 2009 -0500
# Node ID e6a65a8605bc389afd2aef31ab4a192837db293e
# Parent  5a5dbdacbf5d2d8beb869417f55df76772058ca8
src/version.h: append + to version and api

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,10 +1,13 @@
 2009-02-12  John W. Eaton  <jwe@octave.org>
 
+	* version.h (OCTAVE_VERSION): Now 3.1.52+.
+	(OCTAVE_API_VERSION): Now api-v34+.
+
 	* TEMPLATE-INST/Array-tc.cc
 	(octave_sort<octave_value>::ascending_compare,
 	octave_sort<octave_value>::descending_compare):
 	Delete unused template specializations.
 	Use NO_INSTANTIATE_ARRAY_SORT instead of INSTANTIATE_ARRAY_SORT
 	for octave_values.
 
 2009-02-11  Thomas Treichl  <Thomas.Treichl@gmx.net>
diff --git a/src/version.h b/src/version.h
--- a/src/version.h
+++ b/src/version.h
@@ -19,19 +19,19 @@ You should have received a copy of the G
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
 
 #if !defined (octave_version_h)
 #define octave_version_h 1
 
-#define OCTAVE_VERSION "3.1.52"
+#define OCTAVE_VERSION "3.1.52+"
 
-#define OCTAVE_API_VERSION "api-v34"
+#define OCTAVE_API_VERSION "api-v34+"
 
 #define OCTAVE_RELEASE_DATE "2008-02-09"
 
 #define OCTAVE_COPYRIGHT "Copyright (C) 2008 John W. Eaton and others."
 
 // This is the first line printed by --version.  The GNU coding
 // standards say that the version number should follow the last space
 // on the line.
